docs/developer.md

```diff
@@ -61,16 +61,15 @@ The development environment uses Docker Compose to create an isolated area for P
 You can easily start it by executing:
 
 ```bash
-$ make start
+$ plz start
 ```
 
 This will create a `mysql` and `vault` container:
  - Vault GUI: http://localhost:8200 login with token found in `cat ~/.vault-token`
 
-**Note:** If you want to customize mount points and port mappings, create a `docker-compose.override.yml` file via
-`make docker-compose.override.yml` and edit the file manually. Please note that you might need to edit the application
-configuration file as well.
+**Note:** If you want to customize mount points and port mappings, create a `docker-compose.override.yml` file via `make docker-compose.override.yml` and edit the file manually. Please note that you might need to edit the application configuration file as well.
```

Comment:
**Question**: is this rewrapping intentional?
What is the reason behind?
(We don't have any strict wrapping rules yet, but if it was automatically made by IDE, we should sync up, because I use a wrap-to-80-chars-long-lines extension for Markdown text.)

Reply:
I haven't noticed that wrap rule so I just assumed it was wrapped that way for no reason and removed it. Now I checked and most of the lines don't break at 80 characters, but it is a good idea to make the line wrapping consistent.

Reply:
Yeah, I haven't "refactored" all instances to incorporate this change on any existing document, it just happened to touched/new content by automation on my end.